docs/api/asyncio_con.rst

```diff
@@ -102,11 +102,11 @@ Connection
 
         >>> import asyncio
         >>> import edgedb
-        >>> async def run():
+        >>> async def main():
```

Comment:
This change is an extra, non-functional addition. IMO, it makes significantly more sense logically to use `asyncio.run()` on a coroutine named *main*, rather than using `asyncio.run()` on a coroutine named *run*.